examples/broadcast/main.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/pion/rtcp"
+	"github.com/pion/webrtc-v3-design"
+	"github.com/pion/webrtc/v3"
+
+	"github.com/pion/webrtc/v3/examples/internal/signal"
+)
+
+const (
+	rtcpPLIInterval = time.Second * 3
+)
+
+func main() {
+	sdpChan := signal.HTTPSDPServer()
+
+	// Everything below is the Pion WebRTC API, thanks for using it ❤️.
+	offer := webrtc.SessionDescription{}
+	signal.Decode(<-sdpChan, &offer)
+	fmt.Println("")
+
+	// Since we are answering use PayloadTypes declared by offerer
+	mediaEngine := webrtc.MediaEngine{}
+	err := mediaEngine.PopulateFromSDP(offer)
+	if err != nil {
+		panic(err)
+	}
+
+	// Create the API object with the MediaEngine
+	api := webrtc.NewAPI(webrtc.WithMediaEngine(mediaEngine))
+
+	peerConnectionConfig := webrtc.Configuration{
+		ICEServers: []webrtc.ICEServer{
+			{
+				URLs: []string{"stun:stun.l.google.com:19302"},
+			},
+		},
+	}
+
+	// Create a new RTCPeerConnection
+	peerConnection, err := api.NewPeerConnection(peerConnectionConfig)
+	if err != nil {
+		panic(err)
+	}
+
+	// Allow us to receive 1 video track
+	//FIXED-TO-V3 TODO
+	if _, err = peerConnection.AddTransceiverFromKind(webrtc.RTPCodecTypeVideo); err != nil {
```

Comment:
```suggestion
	if _, err = peerConnection.AddTransceiverFromKind(webrtc.RTPCodecTypeVideo, nil); err != nil {
```
discussion: https://github.com/pion/webrtc-v3-design/pull/12#discussion_r460467713